assets/js/juliana.js

```diff
@@ -302,6 +319,27 @@ Receipt = {
         var sum = Receipt.updateTotalAmount();
         var amount = Math.ceil(sum / 10) * 10;
         State.toggleTo(State.MESSAGE, 'Dat wordt dan &euro; ' + (amount/100).toFixed(2));
+    },
+    writeoffNow: function (categoryId) {
+        let rpcRequest = {
+            jsonrpc: '2.0',
+            method: 'juliana.writeoff.save',
+            params: {
+                event_id: Settings.event_id,
+                writeoff_id: categoryId,
+                purchases: Receipt.receipt,
+            },
+            id: 2 // id used for?
```

Comment:
Still not sure what this ID is used for.

Reply:
This `id` field is part of the JSONRPC spec, and must be a client-specified identifier for the request.
The current value `2` is probably fine for our use-case.
See also the IA API docs: https://www.inter-actief.utwente.nl/api/docs/#header_jsonrpcrequest